internal/gps/pkgtree/digest_test.go

```diff
@@ -139,6 +58,32 @@ func TestDigestFromDirectory(t *testing.T) {
 	})
 }
 
+func BenchmarkDigestFromDirectory(b *testing.B) {
+	prefix := getTestdataVerifyRoot(b)
+	b.ResetTimer()
+
+	for i := 0; i < b.N; i++ {
+		_, err := DigestFromDirectory(prefix)
+		if err != nil {
+			b.Fatal(err)
+		}
+	}
+}
+
+const flameIterations = 100000
```

Comment:
could you explain the benefit of having a fixed number of iterations, rather than relying (as `BenchmarkDigestFromDirectory()` does) on the testing system to either a) empirically determine an appropriate threshold and/or b) allow the user to control it via CLI args?

Reply:
When comparing flame graphs, I want to compare how many cycles were spent in the corresponding functions for the same input. Those numbers are skewed if we run version A 500 times, and version B only 100 times. Therefore, forcing the iteration count to a particular number ensures an apples to apples comparison.

Reply:
gotcha, wfm 👍